spring-cloud-azure-context/src/main/java/com/microsoft/azure/spring/cloud/context/core/AzureAdmin.java

```diff
@@ -245,6 +248,22 @@ public RedisCache getOrCreateRedisCache(String name) {
         return getOrCreate(this::getRedisCache, this::createRedisCache, RedisCache.class).apply(name);
     }
 
+    private CosmosDBAccount getCosmosDBAccount(Tuple kindAndName){
+        return azure.cosmosDBAccounts().getByResourceGroup(resourceGroup, (String) kindAndName.getSecond());
+    }
+
+    private CosmosDBAccount createCosmosDBAccount(Tuple kindAndName){
+        return azure.cosmosDBAccounts().define((String) kindAndName.getSecond()).withRegion(region)
+                .withExistingResourceGroup(resourceGroup)
+                .withKind((DatabaseAccountKind) kindAndName.getFirst())
+                .withStrongConsistency().withReadReplication(Region.US_EAST).create();
```

Comment:
Is withStrongConsistency required? If yes, then we need user to provide which region the read replication resides on.